packages/contracts-bedrock/test/L1/OPContractsManager.t.sol

```diff
@@ -246,11 +247,18 @@ contract OPContractsManager_Upgrade_Harness is CommonTest {
         }
 
         // Create validationOverrides
+        address challengerOverride;
```

Comment:
It's probably a good idea to add a test that runs the upgrade, and then deploys a couple of games and checks all of the immutable args are as expected.  There is a [test here](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/test/L1/OPContractsManager.t.sol#L1999-L2024) that does this post OPCM.deploy().

Reply:
The StandardValidator contract was updated in this PR to also check the immutable args. Is there a gap in those checks that should be filled in elsewhere?

Reply:
Added a post-upgrade test to do this. Let me know what you think.